### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by KetanGadwale**
Exercise 2.1 and Exercise 2.2

### CODE REVIEW

File: Week-2/Exercise 2.1/css/style.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+nav a {
+    color: white;
+}
+
+nav>ul {
+    display: flex;
+    gap: 3rem;
+}
+
+ul>li:hover {
```
Comment (GouthamNT) [inline Week-2/Exercise 2.1/css/style.css:59]: space between
ul > li: hover

File: Week-2/Exercise 2.1/css/style.css [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+}
+
+section.articles {
+    margin-top: -8rem;
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 1fr;
+}
+
+article {}
```
Comment (GouthamNT) [inline Week-2/Exercise 2.1/css/style.css]: Never put up empty selectors

File: Week-2/Exercise 2.1/css/style.css (hunk 1 above)
Comment (KetanGadwale) [reply Week-2/Exercise 2.1/css/style.css:59]: Code formatter (ESLint) was removing the spaces. Will try to figure this out.

File: Week-2/Exercise 2.1/css/style.css (hunk 2 above)
Comment (KetanGadwale) [reply Week-2/Exercise 2.1/css/style.css]: Forgot to remove it. Will make sure that this won't happen again.

**Pull request merged**